tests/testSdssShape.py

```diff
@@ -127,6 +189,7 @@ def suite():
     suites = []
     suites += unittest.makeSuite(SdssShapeTestCase)
     suites += unittest.makeSuite(SdssShapeTransformTestCase)
+    suites += unittest.makeSuite(PsfSdssShapeTransformTestCase)
```

Comment:
This might be a good time to update this test to the [new unit test boilerplate](https://developer.lsst.io/coding/python_testing.html#memory-and-file-descriptor-leak-testing).